languages/c/reference/exercise-concepts/two-fer.md

```diff
@@ -0,0 +1,22 @@
+# Concepts of two-fer
+
+```c
+#include "two_fer.h"
+#include <stdio.h>
+
+void two_fer(char *buffer, const char *name) {
+    sprintf(buffer, "One for %s, one for me.", name ? name : "you");
+}
+```
+
+- [Header Files](https://www.gnu.org/software/libc/manual/html_mono/libc.html#Header-Files): used to access the standard library
+- [C Standard Library header files](https://en.cppreference.com/w/c/header): stdio.h is a C standard library header file
+- [Function Definitions](https://www.gnu.org/software/gnu-c-manual/gnu-c-manual.html#Function-Definitions): used as the main entry point for the exercise
+- [Function Parameters](https://www.gnu.org/software/gnu-c-manual/gnu-c-manual.html#Function-Parameters): buffer and name are passed as arguments
+- [Integer Types](https://www.gnu.org/software/gnu-c-manual/gnu-c-manual.html#Integer-Types): buffer and name are integer types
```

Comment:
I think the "Void Type" should also be added as a Concept.

Reply:
Thanks, I'll add that, and probably `NULL` as well, since it gets passed into the function when the default name should be used.

Reply:
👍